packs/rackspace/actions/add_node_to_loadbalancer.py

```diff
@@ -0,0 +1,23 @@
+from lib.action import PyraxBaseAction
+
+__all__ = [
+    'AddNodeToLoadBalancerAction'
+]
+
+class AddNodeToLoadBalancerAction(PyraxBaseAction):
+    def run(self, loadbalancer_id, ip, port):
+        clb = self.pyrax.cloud_loadbalancers
+        node = clb.Node(address=ip, port=port, condition="ENABLED")
+
+        self.logger.info('Adding node to loadbalancer...')
+
+        load_balancer = clb.get(loadbalancer_id)
+        load_balancer.add_nodes(node)
+
+        # Block until added
+        self.pyrax.utils.wait_until(load_balancer, "status", "ACTIVE",
```

Comment:
It might make sense to add `async` parameter for those actions which defaults to false. This way user can also run those actions in "async: manner.